src/pages/guide/comparison-with-leading-frameworks.ejs

```diff
@@ -141,7 +141,7 @@ const data = {
         <td>
           <span class="comp-yes">✅</span> <br>
           <p>Mobile web is just as easy to setup and supports everything from Desktop web.
-            <br><a href="#">Get Started</a>
+            <br><a href="https://nightwatchjs.org/guide/mobile-app-testing/introduction.html">Get Started</a>
```

Comment:
This link is still not working in the deploy preview. Not sure why `.html` is not appearing in the hyperlink.

Reply:
There are many places we have referenced like that, not sure how to resolve.